src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -42,6 +42,8 @@ window.bp = window.bp || {};
 			this.postFormView();
 
 			this.postFormPlaceholderView();
+
+			this.checkDraftContentInPostForm();
```

Comment:
@shohelbuddyboss We should make use of Backbone events here like this https://github.com/buddyboss/buddyboss-platform/blob/dev/src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js#L2505

https://github.com/buddyboss/buddyboss-platform/blob/dev/src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js#L2532